library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3226,4 +3226,11 @@
     <string name="device_manager_other_sessions_description_verified">Verified · Last activity %1$s</string>
     <string name="device_manager_other_sessions_description_unverified">Unverified · Last activity %1$s</string>
 
+    <string name="home_empty_space_no_rooms_title">%s \nis looking a little empty.</string>
```

Comment:
Please remove the extra space
```suggestion
    <string name="home_empty_space_no_rooms_title">%s\nis looking a little empty.</string>
```

Also maybe add a comment above to tell the translator that `%s` will be replaced by the space name (IIUC)